website_snippet_country_dropdown/__init__.py

```diff
@@ -1,2 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
-import test
\ No newline at end of file
+from . import tests
```

Comment:
Oh sorry I think I mistaked before in the comment. You don't have to import tests in the addon's `__init__.py`; never.
This module gets autoimported when running the unit tests.